From cc69fb9fce8d4e06a80e419596570b36c08565da Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Wed, 19 Nov 2014 15:52:18 +0800
Subject: [PATCH 1195/1543] MLK-9843-01 arm: mcc: set dual copyright of mcc
 common codes

commit 2cc5d94be791916e00e65a26d0eb63f43f30e1d9 from
git://git.freescale.com/imx/linux-2.6-imx.git

set the dual bsd/gpl copyright of the mcc common codes
implemented in linux bsp release.
add 2015.

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
(cherry picked from commit d7f8c4eff50ba78b82484597bd301c4a87dfe082)
---
 arch/arm/mach-imx/mcc_api.c    |   20 +++++++++-----------
 arch/arm/mach-imx/mcc_common.c |   20 +++++++++-----------
 arch/arm/mach-imx/mcc_config.h |   20 +++++++++-----------
 3 files changed, 27 insertions(+), 33 deletions(-)

diff --git a/arch/arm/mach-imx/mcc_api.c b/arch/arm/mach-imx/mcc_api.c
index 68c9ed0..8814123 100755
--- a/arch/arm/mach-imx/mcc_api.c
+++ b/arch/arm/mach-imx/mcc_api.c
@@ -1,18 +1,16 @@
 /*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
- * Freescale IMX Linux-specific MCC implementation.
+ * This file contains MCC library API functions implementation
  *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
+ * Copyright (C) 2014-2015 Freescale Semiconductor, Inc. All Rights Reserved.
  *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
- * more details.
  *
- * You should have received a copy of the GNU General Public License along with
- * this program. If not, see <http://www.gnu.org/licenses/>.
+ * SPDX-License-Identifier: GPL-2.0+ and/or BSD-3-Clause
+ * The GPL-2.0+ license for this file can be found in the COPYING.GPL file
+ * included with this distribution or at
+ * http://www.gnu.org/licenses/gpl-2.0.html
+ * The BSD-3-Clause License for this file can be found in the COPYING.BSD file
+ * included with this distribution or at
+ * http://opensource.org/licenses/BSD-3-Clause
  */
 
 #include "mcc_config.h"
diff --git a/arch/arm/mach-imx/mcc_common.c b/arch/arm/mach-imx/mcc_common.c
index 7a54d7b..5c864f3 100755
--- a/arch/arm/mach-imx/mcc_common.c
+++ b/arch/arm/mach-imx/mcc_common.c
@@ -1,18 +1,16 @@
 /*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
- * Freescale IMX Linux-specific MCC implementation.
+ * This file contains MCC library common functions
  *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
+ * Copyright (C) 2014-2015 Freescale Semiconductor, Inc. All Rights Reserved.
  *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
- * more details.
  *
- * You should have received a copy of the GNU General Public License along with
- * this program. If not, see <http://www.gnu.org/licenses/>.
+ * SPDX-License-Identifier: GPL-2.0+ and/or BSD-3-Clause
+ * The GPL-2.0+ license for this file can be found in the COPYING.GPL file
+ * included with this distribution or at
+ * http://www.gnu.org/licenses/gpl-2.0.html
+ * The BSD-3-Clause License for this file can be found in the COPYING.BSD file
+ * included with this distribution or at
+ * http://opensource.org/licenses/BSD-3-Clause
  */
 
 #include "mcc_config.h"
diff --git a/arch/arm/mach-imx/mcc_config.h b/arch/arm/mach-imx/mcc_config.h
index 47abf79..bae8faf 100755
--- a/arch/arm/mach-imx/mcc_config.h
+++ b/arch/arm/mach-imx/mcc_config.h
@@ -1,18 +1,16 @@
 /*
- * Copyright (C) 2014 Freescale Semiconductor, Inc.
- * Freescale IMX Linux-specific MCC implementation.
+ * This is the main MCC configuration file
  *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms and conditions of the GNU General Public License,
- * version 2, as published by the Free Software Foundation.
+ * Copyright (C) 2014-2015 Freescale Semiconductor, Inc. All Rights Reserved.
  *
- * This program is distributed in the hope it will be useful, but WITHOUT
- * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
- * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
- * more details.
  *
- * You should have received a copy of the GNU General Public License along with
- * this program. If not, see <http://www.gnu.org/licenses/>.
+ * SPDX-License-Identifier: GPL-2.0+ and/or BSD-3-Clause
+ * The GPL-2.0+ license for this file can be found in the COPYING.GPL file
+ * included with this distribution or at
+ * http://www.gnu.org/licenses/gpl-2.0.html
+ * The BSD-3-Clause License for this file can be found in the COPYING.BSD file
+ * included with this distribution or at
+ * http://opensource.org/licenses/BSD-3-Clause
  */
 
 #ifndef __MCC_CONFIG__
-- 
1.7.5.4

